index.html

```diff
@@ -0,0 +1,308 @@
+            <li class="dev-tool2"><p>css</p></li>
+            <li class="dev-tool3"><p>javascript</p></li>
+          </ul>
+          <div class="projects-tonic">
+            <button type="submit" class="go-projects">See projects</button>
+          </div>
+        </div>
+      </section>
+
+      <section class="works-card">
+        <img
+          class="snapshort"
+          src="./images/Snapshoot-Portfolio3.png"
+          alt="snapshoot"
+        />
+        <div class="left-block">
+          <div class="primary-container">
+            <h1 class="project-title">
+              Multi-Post <br />
+              Stories
+            </h1>
+            <div class="frame2">
+              <div class="client"><h3>CANOPY</h3></div>
+              <img src="./images/Counter.png" alt="counter image" />
+              <div class="role"><h3>Back End Dev</h3></div>
+              <img src="./images/Counter.png" alt="counter image" class="dot" />
+              <div class="year"><h3>2015</h3></div>
+            </div>
+          </div>
+          <p class="project-description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="dev-tools">
+            <li class="dev-tool1"><p>html</p></li>
+            <li class="dev-tool2"><p>css</p></li>
+            <li class="dev-tool3"><p>javascript</p></li>
+          </ul>
+
+          <div class="projects">
+            <button type="submit" class="go-projects">See projects</button>
+          </div>
+        </div>
+      </section>
+    </main>
+    <section class="contact-me">
+      <div class="headline-section-co">
+        <div id="headline-container-co">
+          <h1 class="headline-co">
+            About, <br />
+            Myself
+          </h1>
+          <p class="description-co">
+            Hello I’m a software developer! I can help you build a product,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h2 class="connect-co">LET’S CONNECT</h2>
+
+          <ul class="social-media-icons">
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/angelite.png" alt="image" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank"> <img src="images/twitter.png"  alt="image"/> </a>
+            </li>
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/Linkedin.png" alt="image" />
+              </a>
+            </li>
+            <li>
+              <a href="#" target="_blank"> <img src="images/medium.png" alt="image" /> </a>
+            </li>
+            <li>
+              <a href="#" target="_blank"> <img src="images/github.png" alt="image"/> </a>
+            </li>
+          </ul>
```

Comment:
- [ ] `required`
- [ ] :wave: you miss placed the social icons kindly put them in place. see screenshots below


`figma template`
![Screenshot from 2022-07-06 09-31-54](https://user-images.githubusercontent.com/66230306/177637930-e3b5a1da-e4b7-4311-a5ad-632d85bee124.png)


`Your template`
![Screenshot from 2022-07-06 09-30-27](https://user-images.githubusercontent.com/66230306/177638281-8a0057b1-ac95-4023-b220-ad383020e56c.png)